### PR TITLE
Component outdated warnings

### PR DESCRIPTION
**Pull request by olemoi**

### CODE REVIEW

File: src/components/environments-summary/environment-ingress.js [hunk 1]
```diff
@@ -1,40 +1,99 @@
...
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
 
 import usePollComponents from './use-poll-components';
 
 const URL_VAR_NAME = 'RADIX_PUBLIC_DOMAIN_NAME';
-const MAX_DISPLAY_NR_COMPONENT = 3;
+const MAX_DISPLAY_NR_COMPONENT = 4;
+
+const outdatedOrFailedComponent = (component) => {
+  console.log(component);
```
Comment (nilsgstrabo) [inline src/components/environments-summary/environment-ingress.js:12]: Remove console.log

File: src/components/environments-summary/environment-ingress.js [hunk 2]
```diff
@@ -1,40 +1,98 @@
...
+    passiveComponents.length > MAX_DISPLAY_NR_COMPONENT;
+
   if (tooManyPublicComponents) {
-    components = components.slice(0, MAX_DISPLAY_NR_COMPONENT);
+    publicComponents = publicComponents.slice(0, MAX_DISPLAY_NR_COMPONENT);
+  }
+
+  if (tooManyPassiveComponents) {
+    passiveComponents = passiveComponents.slice(0, MAX_DISPLAY_NR_COMPONENT);
   }
```
Comment (nilsgstrabo) [inline src/components/environments-summary/environment-ingress.js:68]: Would it be better to use one slice for borth public and passive components by appending the passive (only those failing and outdated) to the public slice.
Currently, if there are four public components and some passive components that are failing, the height of the comp-list would exceed the environment box.
Using one slice also reduces code duplication.

File: src/components/environments-summary/environment-ingress.js [hunk 3]
```diff
@@ -1,40 +1,98 @@
...
+              <a href={`https://${component.variables[URL_VAR_NAME]}`}>
+                {component.name} <FontAwesomeIcon icon={faLink} size="lg" />
+              </a>
+              {outdatedOrFailedComponent(component)}
+            </div>
           </li>
         ))}
-        {tooManyPublicComponents && <li>...</li>}
+        {passiveComponents.map(
+          (component) =>
```
Comment (nilsgstrabo) [inline src/components/environments-summary/environment-ingress.js:84]: Ref comment about using one slice would remove the need for two <slice>.map(...) iterations

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Added a few comments that should be looked at.
Also, consider updating dev.js in environment-summary with examples of the new status

File: src/components/environments-summary/environment-ingress.js (hunk 2 above)
Comment (olemoi) [reply src/components/environments-summary/environment-ingress.js:68]: As discussed, combining the slices could result in none of the failing components to show in the list. Reduced number of max components to account for layout issues.

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**